rdr_service/offline/bigquery_sync.py

```diff
@@ -5,7 +5,7 @@
 from datetime import datetime
 
 from rdr_service.config import GAE_PROJECT
-from google.appengine.api import taskqueue
+#from google.appengine.api import taskqueue @todo
```

Comment:
I added a placeholder for this in #1197
you can drop the changes in this file from this PR

Reply:
done